src/prognostic_equations/surface_flux.jl

```diff
@@ -21,9 +42,27 @@ function boundary_tendency_momentum(ᶜρ, ᶜuₕ, ρ_flux_uₕ)
 end
 
 """
-    Boundary tendency for scalars, applied through the divergence operator
-    with zeros in the interior and user specified boundary values. The
-    resulting tendency is only non-zero at the boundaries.
+    boundary_tendency_scalar(ᶜχ_field_template, ρ_flux_χ_surface)
+
+Calculates the tendency contribution for a scalar quantity `χ` (for the prognostic 
+variable `ρχ`) due to a specified vertical flux of that scalar at the bottom boundary.
+
+This function constructs a divergence term that is non-zero only in the grid cell
+adjacent to the bottom boundary, effectively introducing `ρ_flux_χ_surface` as a
+source/sink. When positive, the flux `ρ_flux_χ_surface` is directed from the surface to 
+the atmosphere, i.e., represents an atmospheric source.  
+
+Arguments:
+- `ᶜχ_field_template`: A cell-center scalar field of the same type as the quantity
+  `χ` for which the tendency is being computed (used for eltype and spatial structure,
+  not its values in the flux calculation).
+- `ρ_flux_χ_surface`: The vertical flux of the scalar quantity `χ` (density-weighted,
+  i.e., flux of `ρχ`) through the bottom boundary. This is a
```

Comment:
```suggestion
- `ᶜχ`: cell-center scalar field (used for eltype and spatial structure,
  not its values in the flux calculation).
- `ρ_flux_χ`: The vertical flux of the scalar quantity `χ` (density-weighted,
  i.e., flux of `ρχ`) through the bottom boundary. This is a
```
(or change the function argument to ρ_flux_χ_surface)